### PR TITLE
CI change, only run bench on performance label

### PR DESCRIPTION
**Pull request by soulomoon**
We already have the `performance` label

### CODE REVIEW

File: .github/workflows/bench.yml [hunk 1]
```diff
@@ -17,6 +17,7 @@ on:
 jobs:
   pre_job:
     runs-on: ubuntu-latest
+    if: contains(github.event.pull_request.labels.*.name, 'performance')
```
Comment (soulomoon) [inline .github/workflows/bench.yml:20]: I am not quite sure if this is enough ?

**Comment by fendor** [comment]
CI looks like it skipped the benchmark job.

To we want to make nightly runs of the benchmarks perhaps? Ideally, there would be an easy to access them later.

**Comment by soulomoon** [comment]
CI already skipped bench on some occasions.

And we almost never looking at them unless we touch performance related code.

I think for now, if we touch performance related, manually adding the performance label should be enough.

But we can still add the nightly runs, if someone wants it. I want to know which branch would the nightly runs be running, for all PR or just the mater?

**Review by fendor — APPROVED** [review]
LGTM

**Pull request merged**